repo/packages/L/linkerd-viz/3/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "packagingVersion": "3.0",
+  "name": "linkerd-viz",
+  "version": "0.1.0-0.1",
+  "scm": "https://github.com/linkerd/linkerd-viz/",
+  "description": "A visualization dashboard for linkerd.",
+  "maintainer": "linkerd-users@googlegroups.com",
+  "website": "https://linkerd.io",
+  "tags": ["linkerd", "namerd", "buoyant", "loadbalancer", "service-discovery", "proxy", "microservices", "rpc", "linkerd-viz", "dashboard", "visualization", "metrics"],
+  "preInstallNotes": "This DC/OS Service is currently in preview. We recommend a minimum of 1.0 CPU and 512 MB of RAM available for linkerd-viz.",
```

Comment:
add a link to dcos/examples at the end of `preInstallNotes`. 
Also, please update the example to add 1.9 instructions (see other examples): https://github.com/dcos/examples/tree/master/linkerd